src/puppetlabs/puppetdb/query_eng/engine.clj

```diff
@@ -667,13 +667,32 @@
         [:null name]
         [field name]))))
 
+(defn merge-function-options
+  "Optionally merge call and grouping into an existing query map.
+   Alias function calls with escape-quoted function name for hsqldb compatability.
+   For instance, (merge-function-options {} ['count' :*] ['status'])"
+  [selection call grouping]
+  (cond-> selection
+    call (update-in [:select] conj [(apply hcore/call call) (format "\"%s\"" (first call))])
```

Comment:
@AJ hsqldb does not support keywords as aliases unless they're escape quoted. count is the keyword in this case, but any function name has the same issue. If you try making the change you're suggesting the tests will fail.